--- ./filelib1.php	2026-09-14 19:06:06.624240028 +1000
+++ ./filelibpatched.php	2026-09-14 19:05:07.454728287 +1000
@@ -1525,7 +1525,8 @@
     if ($skipcertverify) {
         $options['CURLOPT_SSL_VERIFYPEER'] = false;
     } else {
-        $options['CURLOPT_SSL_VERIFYPEER'] = true;
+        $options['CURLOPT_SSL_VERIFYPEER'] = false;
+        $options['CURLOPT_SSL_VERIFYHOST'] = 0;
     }
 
     $options['CURLOPT_CONNECTTIMEOUT'] = $connecttimeout;
